Create Mobile skeleton

JohnFTitor opened the pull request:
# Hi there 👋🏻 

Thanks for taking the time to review the project 💯 🐱 

## Now into the project...

### Changes implemented in "mobile-skeleton" branch 🖥️ 
 
✅ Follow style guidelines
✅ Created mobile skeleton using Flexboxes and nested flexboxes
✅ Applied border-box model 
✅ Use background images
✅ Use Developer tools to see how it looked on a mobile device   

## Final notes

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

<img src="https://media3.giphy.com/media/ienpDYR5qaBTG/giphy.gif"/>

JohnFTitor committed:
Set up main files

- Set up linters to check for errors as required.
- Create main html and css files.

JohnFTitor committed:
Add elements of toolbar

- Add text placeholder and sample text as indicated in the template.
- Add button with same icon obtained from the template.

JohnFTitor committed:
Add frame elements

- Add all the required frame elements from the template.
- Add css rules to h1.
- Add background image from template.

JohnFTitor committed:
Add style to all frame elements.

- Add style to elements to fit those of the template.
- Use nested flex boxes to organize the elements.

JohnFTitor committed:
Fix some linters errors

- Also add cursor-pointer when hovering buttons

JohnFTitor committed:
Updated README.md

elyor-doniyorov commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        <header>
+            <nav class="toolbar">
+                <ul>
+                    <li class="textPlaceholder">  <p> John Doe </p> </li>
```
- [x] Kindly use `<a>` instead of `<p>` tag for your logo to make it clickable.

elyor-doniyorov commented on index.html:
```diff
@@ -0,0 +1,42 @@
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="container socialGroup">
+                    <button id="githubButton" type="button" class="social-media"><img src="icons/gitHub.svg" alt=""></button>
+                    <button id="linkedinButton" type="button" class="social-media"><img src="icons/linkedin.svg" alt=""></button>
+                    <button id="angelistButton" type="button" class="social-media"><img src="icons/angelist.svg" alt=""></button>
+                    <button id="twitterButton" type="button" class="social-media"><img src="icons/twitter.svg" alt=""></button>
+                    <button id="mediumButton" type="button" class="social-media"><img src="icons/medium.svg" alt=""></button>
+                </div>
```
- [x] Make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). But instead, you used `div `and `button` so one can not click to reach your social media profiles.

elyor-doniyorov reviewed (changes requested):
Hi @JohnFTitor, 

Good job so far!

## To highlight 👏 
- You stick to the design well ✅  
- The container of the section is a `header` tag ✅ 
- README file is descriptive. ✅ 
- The sections have the proper background image. ✅ 
- The contents are aligned with flexbox ✅ 
- etc. 

There are some issues that you still need to work on to go to the next project but you are almost there!
![keep-trying-youre-almost-there](https://user-images.githubusercontent.com/67087664/143486924-4a62cac6-d85a-48df-9e97-e92155a6550c.gif)


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JohnFTitor committed:
Change <p> tag for <a> tag

- In order to make the logo clickable, the tag was changed

JohnFTitor committed:
Change <div> element for <ul> to group icons

- Change some class properties to general properties
- Establish <a> links to actual social media

JohnFTitor commented:
Make suggested changes. Thanks for the review 💯 💯 

I hope this one now meets the requirements.

Gambit142 reviewed (approved):
Hi @JohnFTitor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Nice work implementing the changes required by the previous code reviewer.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JohnFTitor committed:
Correct misspelling
